src/Context/Context.php

```diff
@@ -19,15 +16,15 @@ final class Context
 
     // Optimization for spans to avoid copying the context array.
     private static ContextKey $spanContextKey;
-    private ?SpanInterface $span = null;
+    private ?object $span = null;
```

Comment:
`$span` is probably no longer the right name for this variable, it looks like it can be anything?

Reply:
Still the correct name, this property holds the value that is assigned to the span context key.